guide/ops/troubleshooting/deployment.md

```diff
@@ -143,3 +152,27 @@ start tasks are completed).
 
 See the [overview](overview.html) for where to find additional information, especially the section on
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM (typically the OS default user).
+
+{% highlight bash %}
+error acquiring SFTPClient() (out of retries - max 50)
+Invalid packet: indicated length too large
+java.lang.IllegalStateException
+Invalid packet: indicated length too large
+{% endhighlight %}
+
+An example of how to explicitly set the user is shown below (when defining a Location) by using 'loginUser': 
```

Comment:
I'd add something like:

    In most clouds, the initial `loginUser` depends on the VM image. To fix the error above, 
    identify the correct username for the image and supply this explicitly using the config `loginUser`: